diff --git a/macro/ShipReco.py b/macro/ShipReco.py
index 36867b3..e18c963 100644
--- a/macro/ShipReco.py
+++ b/macro/ShipReco.py
@@ -35,9 +35,22 @@ import shipRoot_conf
 
 shipRoot_conf.configure()
 
+
+# The following lines are used to import from a parent directory
+import inspect
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir)
+
+
+from tue_conditionsDb import api
+
+# Create an instance of our ConditionsDB API
+conditionsDB_API = api.API()
+
 try:
-        opts, args = getopt.getopt(sys.argv[1:], "o:D:FHPu:n:f:g:c:hqv:sl:A:Y:i:",\
-           ["ecalDebugDraw","inputFile=","geoFile=","nEvents=","noStrawSmearing","noVertexing","saveDisk","realPR=","withT0"])
+        opts, args = getopt.getopt(sys.argv[1:], "o:D:FHPu:n:f:g:c:hqv:sl:A:Y:i:t:",\
+           ["ecalDebugDraw","inputFile=","geoFile=","nEvents=","noStrawSmearing","noVertexing","saveDisk","realPR=","withT0","globalTag="])
 except getopt.GetoptError:
         # print help information and exit:
         print ' enter --inputFile=  --geoFile= --nEvents=  --firstEvent=,'
@@ -52,6 +65,15 @@ for o, a in opts:
             withNoStrawSmearing = True
         if o in ("--withT0",):
             withT0 = True
+        if o in ("-t", "--globalTag=", ):
+            globalTag = a
+            print globalTag
+            print '-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-'
+            conditions = conditionsDB_API.get_data_global_tag(globalTag)
+            print conditions
+            for i in conditions:
+                print i.name
+            print '-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-'
         if o in ("-f", "--inputFile",):
             inputFile = a
         if o in ("-g", "--geoFile",):
